bref

```diff
@@ -19,10 +19,10 @@ if (file_exists(__DIR__ . '/vendor/autoload.php')) {
 $app = new Silly\Application('Deploy serverless PHP applications');
 
 $app->command('init', function (SymfonyStyle $io) {
-    $exeFinder = new \Symfony\Component\Process\ExecutableFinder();
-    if ($exeFinder->find('serverless') === null) {
+    $exeFinder = new ExecutableFinder();
+    if (! $exeFinder->find('sam')) {
         $io->error(
-            'The serverless command is not installed.'.PHP_EOL.
+            'The `sam` command is not installed.' . PHP_EOL .
             'Please follow the instructions on https://github.com/mnapoli/bref#setup'
```

Comment:
Should this url link to the file within the same bref version instead of always-master?

Reply:
This URL will be invalid anyway in 0.3 since I'm rewriting the documentation entirely (currently working on that) so I didn't bother here.